File: javascript/fixedheapdynamic.js
function fixedHeapDynamicJS() {
    let heapArray = new Array(5);
    for (let i = 0; i < heapArray.length; i++) {
        heapArray[i] = i * 10;
    }
    console.log(heapArray);
}
fixedHeapDynamicJS();

File: java/fixedheapdynamic.java
public class FixedHeapDynamicJava {
    public static void main(String[] args) {
        Integer[] heapArray = new Integer[5];
        for (int i = 0; i < heapArray.length; i++) {
            heapArray[i] = i * 10;
        }
        for (int val : heapArray) {
            System.out.println(val);
        }
    }
}